### PR TITLE
[Finishes #101465756, #101453700] Chrome-less share for App Lab

### PR DESCRIPTION
**Pull request by joshlory**
This change implements chrome-less App Lab share per @sfilman's spec.

On mobile, chrome-less share shows no Run or Reset button.  The app takes up the full width of the phone, with space left at the bottom (due to the aspect ratio).

screen shot 2015-09-30 at 4 34 21 pm

On desktop, chrome-less share shows no Run or Reset button.  The app is framed by a phone wireframe mock.

screen shot 2015-10-01 at 11 32 01 am

### CODE REVIEW

File: dashboard/app/assets/javascripts/small_footer.js.jsx [hunk 1]
```diff
@@ -179,7 +179,7 @@ window.dashboard.footer = (function () {
               }}/>
               <small>
                 {this.renderCopyright()}
-                <a className="more-link" href="#"
+                <a className="more-link" href="javascript:void(0)"
```
Comment (joshlory) [inline dashboard/app/assets/javascripts/small_footer.js.jsx:182]: [Finishes #104218742]

Comment (Bjvanminnen) [inline dashboard/app/assets/javascripts/small_footer.js.jsx:182]: hah. I'd gone back and forth on using # vs. javascript:void(0). I thought I saw SO answer saying # was better, but it's been problematic in various ways. I wonder if we should be doing a broader cleanup of these.

**Comment by Bjvanminnen** [comment]
Pretty cool.  Lgtm

**Comment by sfilman** [comment]
Hey Josh, is the asset not sized quite correctly? Ideally we don't have the
white border around the footer and instead that butts up against the phone
border:
[image: Inline image 1]

Let me know if so and i can tweak.

**Comment by sfilman** [comment]
Also (this looks so cool, btw!) can we push the phone down by 50 pixels so
it's not touching the browser bar?

**Pull request merged**